requirements-dev.txt

```diff
@@ -7,6 +7,7 @@ pytest~=8.0.0
 pytest-benchmark
 pytest-cov
 pytest-mock
+pytest-xdist
```

Comment:
Why are we adding this?

Reply:
It may have slipped there after switching branches. It will be used to accelerate native MCM tests in #621 .